inbox/events/timezones.py

```diff
@@ -107,11 +110,160 @@
     'W. Europe Standard Time': 'Europe/Berlin',
     'West Asia Standard Time': 'Asia/Tashkent',
     'West Pacific Standard Time': 'Pacific/Port_Moresby',
-    'Yakutsk Standard Time': 'Asia/Yakutsk',
+    'Yakutsk Standard Time': 'Asia/Yakutsk'
 }
 
+# Also add the friendly timezone names
+
+utc_friendly_timezones = {
```

Comment:
Hmm leading caps in local vars OK? What's the standard python approach? Really don't care.